releasenotes-dca/notes/fix-annotation-04d42b4fe4ee4d7f.yaml

```diff
@@ -0,0 +1,4 @@
+---
+security:
+  - |
+    Fix the removal of the "kubectl.kubernetes.io/last-applied-configuration" annotation on new collected resources
```

Comment:
if the bug was introduced in 1.17 and was never released we don't have to add a release note